### PR TITLE
Bugfix: fix EI report calculated values

### PR DESCRIPTION
**Pull request by BCerki**
card: https://github.com/bcgov/cas-cif/issues/1383

### CODE REVIEW

File: schema/deploy/computed_columns/form_change_calculated_ei_performance.sql [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+returns numeric
+as
+$fn$
+
+  select
+    case
+      when (fc.new_form_data->>'baselineEmissionIntensity' is null)
+        or (fc.new_form_data->>'targetEmissionIntensity' is null)
+        or (fc.new_form_data->>'postProjectEmissionIntensity' is null)
+        or (fc.new_form_data->>'baselineEmissionIntensity' = fc.new_form_data->>'targetEmissionIntensity')
```
Comment (gurjmatharu) [inline schema/deploy/computed_columns/form_change_calculated_ei_performance.sql:15]: should the case `or (fc.new_form_data->>'baselineEmissionIntensity' = fc.new_form_data->>'targetEmissionIntensity')` return a non-none value?

Comment (BCerki) [reply schema/deploy/computed_columns/form_change_calculated_ei_performance.sql:15]: If baseline and target are equal, that produces dividing by 0, and I've always seen this handled using `null`. Is there something else that's better to return?

Comment (pbastia) [reply schema/deploy/computed_columns/form_change_calculated_ei_performance.sql:15]: Should we test
```
(fc.new_form_data->>'baselineEmissionIntensity')::numeric - (fc.new_form_data->>'targetEmissionIntensity')::numeric) != 0
```
instead?

**Review by pbastia — DISMISSED** [review]

**Review by pbastia — DISMISSED** [review]

**Review by pbastia — APPROVED** [review]
Very nice!

**Pull request merged**